README.md

```diff
@@ -2,6 +2,16 @@
 
 Work in progress standard library for Lean 4. This is a collection of data structures and tactics intended for use by both computer-science applications and mathematics applications of Lean 4.
 
+# Using `std4`
+
+To use `std4` in your project, add the following to your `lakefile.lean`:
+
+```lean
+require std from git "git@github.com:leanprover/std4.git" @ "main"
```

Comment:
I think this only works for users that use ssh. I think this works for everyone:
```suggestion
require std from git "https://github.com/leanprover/std4" @ "main"
```